app/api_topologies.go

```diff
@@ -21,6 +21,14 @@ var (
 	topologyRegistry = &registry{
 		items: map[string]APITopologyDesc{},
 	}
+	pseudoFilter = APITopologyOptionGroup{
+		ID:      "pseudo",
+		Default: "show",
+		Options: []APITopologyOption{
+			{"show", "Show pseudo nodes", nil, false},
+			{"hide", "Hide pseudo nodes", render.IsNotPseudo, true},
```

Comment:
Also, the filtering area is already pretty crowded, #1129 is not about psuedo nodes per se but about showing groups of pseudonodes unconnected from the rest of the topology. See https://github.com/weaveworks/scope/issues/1129#issuecomment-192688819